js/building.js

```diff
@@ -1,32 +1,44 @@
 class Building{
-    constructor(hp, x, y, game){
-        console.log(game);
+    constructor(hp, x, y, Perseus){
         this.hp = hp;
         this.x = x;
         this.y = y;
-        this.game = game;
+        this.Perseus = Perseus;
+        this.game = Perseus.game;
         this.building = false;
         this.current = null;
         this.units = {};
         this.movable = false;
-        
     }
 
-    
     addSprite(x, y, buildingType){        
         this.sprite = this.game.add.sprite(x, y, buildingType);
+        this.sprite.anchor.x = 0.5;
```

Comment:
Sprites should be anchored around the midpoint like so, in order to simplify drawCircle and selection boxes.